client/src/Component/MainTable.js

```diff
@@ -34,86 +40,146 @@ const MainTable = (props) => {
+											<svg
+												xmlns="http://www.w3.org/2000/svg"
+												width="16"
+												height="16"
+												fill="currentColor"
+												viewBox="0 0 16 16"
+												onClick={() => {
+													props.showOnClientInfoClick(i);
+												}}
+												cursor="pointer"
+											>
+												{props.isHiden === i
+													? arrowPositionDown
+													: arrowPositionUp}
+											</svg>
+										</div>
+										<span>{client.name}</span>
+									</td>
+									<td>{client.phone_number}</td>
+									<td className="align-bikes-needed">{client.bikes_needed}</td>
+									<td className="fs-5">
+										<span className={bookingStatusStyle[client.booking_status]}>
+											{bookingStatus[client.booking_status]}
+										</span>
+									</td>
+									<td>{validDate(client.pick_up_date)}</td>
+								</tr>
+								<tr>
+									<td
+										colSpan="9"
+										style={
+											props.isHiden === i
+												? { display: "table-cell" }
+												: { display: "none" }
+										}
+									>
+										<ul className="list-group">
+											<li className="list-group-item list-group-item-action hiden-rows-columns">
+												<span className="fw-bold">Residence status |</span>
+												<span> {residencyStatus[client.residency_status]}</span>
+											</li>
+											<li className="list-group-item list-group-item-action">
+												<span className="fw-bold">Country of origin |</span>
+												<span> {client.country_of_origin}</span>
+											</li>
+											<li className="list-group-item list-group-item-action">
+												<span className="fw-bold">Time in Scotland |</span>
+												<span> {client.time_in_scotland}</span>
+											</li>
+											<li className="list-group-item list-group-item-action">
+												<span className="fw-bold">Language spoken |</span>
+												<span> {client.language_spoken}</span>
+											</li>
+											<li className="list-group-item list-group-item-action">
+												<span className="fw-bold">English speaker |</span>
+												<span> {speakEnglish(client.english_speaker)}</span>
+											</li>
+											<li className="list-group-item list-group-item-action">
+												<span className="fw-bold">Skill Level |</span>
+												<span> {client.english_skill_level}</span>
+											</li>
+											<li className="list-group-item list-group-item-action">
+												<span className="fw-bold">Gender |</span>
+												<span> {client.gender}</span>
+											</li>
+											<li className="list-group-item list-group-item-action">
+												<span className="fw-bold">Date of birth |</span>
+												<span> {validDate(client.date_of_birth)}</span>
+											</li>
+											<li className="list-group-item list-group-item-action">
+												<span className="fw-bold">Postcode |</span>
+												<span> {client.postcode} {client.referring_agency}</span>
```

Comment:
Can we move referring agency on to it's own `li` element?

Reply:
That's funny, I saw it and looks strange but I do not understand why, ji, ji, ji.